nucypher/characters/control/specifications/bob.py

```diff
@@ -45,15 +41,15 @@ class Retrieve(BaseSchema):
             type=click.STRING,
             required=False)
     )
-    message_kit = character_fields.UmbralMessageKit(
+    message_kit = character_fields.MessageKit(
         required=True,
         load_only=True,
         click=options.option_message_kit(required=False)
     )
 
-    treasure_map = EncryptedTreasureMap(required=False,
-                                        load_only=True,
-                                        click=options.option_treasure_map)
+    encrypted_treasure_map = EncryptedTreasureMap(required=False,
```

Comment:
This is required now, no? If so, then we need to update the `requred` parameter
 
```suggestion
    encrypted_treasure_map = EncryptedTreasureMap(required=True,
```

**AND**

 make the `option_treasure_map` be required.